tensorflow/python/grappler/auto_mixed_precision_test.py

```diff
@@ -566,6 +571,36 @@ def test_conv_pool(self, mode):
     tol = 5e-3 if mode == 'mkl' else 1e-3
     self.assertAllClose(output_val_ref, output_val, atol=tol, rtol=tol)
 
+  # TODO(benbarsdell): This test has not been tried with MKL.
+  @parameterized.parameters(['cuda'])
+  @test_util.run_deprecated_v1
+  @test_util.disable_xla('This test does not pass with XLA')
+  def test_depthwise_conv2d(self, mode):
```

Comment:
You should skip this test for now until we upgrade to cudnn 8. Alternatively, use tf.sysconfig.get_build_info to conditionally skip if the cudnn version is below 8.